VariantShredding.md

```diff
@@ -126,7 +126,8 @@ The list `element` must be a required group.
 The `element` group can contain `value` and `typed_value` fields.
 The element's `value` field stores the element as Variant-encoded `binary` when the `typed_value` is not present or cannot represent it.
 The `typed_value` field may be omitted when not shredding elements as a specific type.
-When `typed_value` is omitted, `value` must be `required`.
+The `value` field may be omitted when shredding elements as a specific type.
```

Comment:
It is fine to say that one of the fields must be present and that `value` can be omitted (though it is not necessary).

Reply:
Thanks for reviewing. In line 68,  we have the statement  `Both `value` and `typed_value` are optional fields used together to encode a single value.` to serve the overall value shredding. That's why we don't need to mention in different place.

Reply:
I don't think we have such case in the tests. The case we have is:  the group should be required but it's passed in as optional. For example:

```
required group event_type {
      optional binary value;
      optional binary typed_value (STRING);
    }
```